op-chain-ops/deployer/integration_test/apply_test.go

```diff
@@ -29,6 +29,7 @@ participants:
   - el_type: geth
     el_extra_params:
       - "--gcmode=archive"
+      - "--rpc.txfeecap=0"
```

Comment:
I don't see any other modifications to do with `txfeecap` in this PR -- is this change needed?

Reply:
Yes, it's necessary. The default fee cap in Geth is 1ETH, and the test goes over 1ETH when deploying contracts via the OPCM. This causes the transaction to be rejected and the test to fail.